tools/clang/unittests/HLSL/ValidationTest.cpp

```diff
@@ -3829,3 +3830,7 @@ TEST_F(ValidationTest, ValidatePrintfNotAllowed) {
   TestCheck(L"..\\CodeGenHLSL\\printf.hlsl");
 }
 
+TEST_F(ValidationTest, ValidateVkNamespaceNotAllowed) {
+  TestCheck(L"..\\CodeGenHLSL\\vk-namespace.hlsl");
+}
+
```

Comment:
Moving the shader to the directory I've indicated above will remove the need for additions to the ValidationTest.cpp. The shader will be run and tested automatically.

Reply:
Done